### DISCUSSION TITLE
Enable both External and Internal Load Balancers

### DISCUSSION THREAD

**Discussion by opsocomusr** [strimzi/strimzi-kafka-operator#5412] [Q&A] [answered]
Is it possible to enable both external and internal load balancers for the same strimzi cluster? When the annotation to enable internal load balancer is added to the Kafka CR, the external IP is replaced with the internal IP.

**Comment by scholzj** [comment, marked as answer]
You have to use separate listener for each and set the annotations through the listener configuration.
